scout/server/blueprints/variants/controllers.py

```diff
@@ -90,19 +90,22 @@ def variants(
 
         evaluations = []
         is_research = variant_obj["variant_type"] == "research"
-        # Get previous ACMG evalautions of the variant from other cases
+        # Get previous ACMG evaluations of the variant from other cases
         for evaluation_obj in store.get_evaluations(variant_obj):
             if evaluation_obj["case_id"] == case_obj["_id"]:
                 continue
 
             classification = evaluation_obj["classification"]
 
-            if is_research or classification not in ["pathogenic", "likely_pathogenic"]:
-                evaluation_obj["classification"] = ACMG_COMPLETE_MAP.get(classification)
-                evaluations.append(evaluation_obj)
+            evaluation_obj["classification"] = ACMG_COMPLETE_MAP.get(classification)
```

Comment:
Oh no, nice fix! 💯

Reply:
> 

Yes, or return the Counter result directly as an int since I'm not storing any other properties from the events anyway. I'll try the list version if it doesn't work.